Doppler.HtmlEditorApi/Infrastructure/Repository.cs

```diff
@@ -25,38 +25,27 @@ public async Task<CampaignContent> GetCampaignModel(string accountName, int camp
 WHERE co.IdCampaign = @campaignId  AND u.Email = @accountName AND co.EditorType = 5";
                 var result = await connection.QueryFirstOrDefaultAsync<ContentRow>(databaseQuery, new { campaignId, accountName });
```

Comment:
Es un cambio mínimo pero por ahí mas compacto... el resultado no se puede retornar de inmediato?
```suggestion
                return await connection.QueryFirstOrDefaultAsync<ContentRow>(databaseQuery, new { campaignId, accountName });
```

Reply:
Buena sugerencia. Corregido.